app/javascript/react/components/SessionFilters/TagsInput.tsx

```diff
@@ -0,0 +1,146 @@
+import React, { useEffect, useState } from "react";
+
+import { useCombobox } from "downshift";
+import { useTranslation } from "react-i18next";
+import { useAppDispatch, useAppSelector } from "../../store/hooks";
+import { setLoading } from "../../store/mapSlice";
+import { fetchTags, selectTags } from "../../store/sessionFiltersSlice";
+import { SessionTypes, fetchTagsParamsType } from "../../types/filters";
+import { UrlParamsTypes, useMapParams } from "../../utils/mapParamsHandler";
+import { FilterInfoPopup } from "./FilterInfoPopup";
+import * as S from "./SessionFilters.style";
+
+const TagsInput = () => {
+  const [items, setItems] = useState([""]);
+  const [inputValue, setInputValue] = useState("");
+  const [selectedItem, setSelectedItem] = useState("");
```

Comment:
Sam suggestions as in `ProfileNamesInput` - adding type annotations would increase safe typing. And `setSelectedItem` is not used anywhere, what are your plans with it?